source/2017-04-13-how-we-got-here.html.markdown.erb

```diff
@@ -0,0 +1,147 @@
+---
+title: Bandwith, coverage, monitoring and other types of Voodoo
+date: 2017-04-13 13:37 UTC
+author: stb
+tags:
+---
+
+As a Sysadmin at Panter you’re also responsible for the Infrastructure of Impact
+Hub Zürich a vibrant coworkingspace in the heart of zürich. On 3 floors and a
+total of roughly 2000 square meters, we’re making sure that all the members fit into our WiFi
+network, as always in such a crowded place there’s not much margin for errors.
+And because our Frequencies are almost as crowded with rogue WiFis like the death
+star’s having holes for thermonuclear missiles, this can be quite challenging and
+sometimes quite frustrating.
+
+[![Youtube:Steve Jobs 500 WiFi Basestations](jobs.jpg)]
+(https://youtu.be/1-oIL9cLHDc?t=19 "Steve Jobs WiFi")
+
+In this Blog post we'd like to give you a little bit of insight into our Configuration
+and how we're troubleshooting our Network. Our WiFi runs exclusively on Mikrotik
+devices because this gives us a lot of flexibility, their devices come at a fair
+price and we're not forced to pay fees to use their products.
+When we first got here we had nothing but an empty Rack, an old DIY Router and
+some broken cables, step by step we've built up the infrastructure adjusting it to our needs.
+
+## The dawn of WiFi
+![Shabby AP](ShabbyAP.jpg)
+
+We started with an emergency setup (2 access points) because we had to move on
+faster than expected, there was not much of a concept, but it worked for us Panters
+which were pretty much alone at that time. While we were having our office on a
+construction site which was quite dusty at the time, we even managed to nearly
+kill our Router because he suffocated in the dust (the whole rack was accidentally
+covered in dust) but it was nothing a little bit of compressed air couldn't solve.
+
+![Bunch of Cables](CableBunch.jpg)
+
+Now working on a construction site has its downsides, but it also gives you the
+freedom to plan your LAN outlets and whatnot, or so we thought, but everything got built quite
+differently as planned so those outlets are still a vision. We had to reverse engineer
+every single one of a bunch of cables because documentation of the existing Network was nonexistent.
+
+When we moved here, our Rack was neat and clean and there were almost no cables,
+we also had to install the cables by ourselves because our top floor was a dusty
+attic which looked like it hasn't been touched for the last half of the decade
+it didn't have any cabling at all. Since then all visible and non-visible LAN
+outlets were patched and if you spot an outlet at our Colab feel free to use it.
+
+![Before and after](beforeAfter.jpg)
+
+With the help of our Network crack (koa) we've got a solid setup and started to
+use Mikrotiks Capsman, which is a bit of a pain to get into but it works like a
+charm after the initial Setup (and 10 Factory Resets later) and a little bit of
```

Comment:
"and a little bit of a hassle to wrap your head around it." should be after "a bit of a pain to get into "